worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/adapter/BukkitImplAdapter.java

```diff
@@ -106,7 +106,15 @@ default void tickWatchdog() {
      * @param location the location
      * @return the block
      */
-    BaseBlock getBlock(Location location);
+    BlockState getBlock(Location location);
+
+    /**
+     * Get the block at the given location.
+     *
+     * @param location the location
+     * @return the block
+     */
+    BaseBlock getFullBlock(Location location);
```

Comment:
How about `#fullBlock` to not use a getter here?

Reply:
This is from upstream